### ISSUE TITLE
Decrease learning rate

### ISSUE THREAD

**Issue by marconardelli** [tensorflow/agents#294]
Hello!
I'm using DQN with Huber loss, looking at tensorboard, I've seen that the loss decrease sharply in the first few iterations and then increases constantly until a certain point.
I guess this is due to the learning rate value. Is there a way to dynamically change the learning rate based on the value of the loss?

**Comment by kbanoop** [comment]
A few points:
1. I found that using a mean squared loss was more stable than huber loss in some environments (not sure why, purely anecdotal)
2. The loss going up is not necessarily a bad thing. As the agent gets better, the QValues start increasing. The loss measures the error in predicting Q Values. Since both the predicted and actual Q values go up, it is possible that the loss also increases.
3. To get a dynamic learning rate, you could do something like: https://www.tensorflow.org/api_docs/python/tf/compat/v1/train/exponential_decay

Closing for now, please reopen if you have more questions
